gobustervhost/gobustervhost.go

```diff
@@ -109,7 +121,13 @@ func (v *GobusterVhost) Run(word string) ([]libgobuster.Result, error) {
 // ResultToString is the to string implementation of gobusterdir
 func (v *GobusterVhost) ResultToString(r *libgobuster.Result) (*string, error) {
 	buf := &bytes.Buffer{}
-	if _, err := fmt.Fprintf(buf, "Found: %s\n", r.Entity); err != nil {
+
+	statusText := "Found"
+	if v.globalopts.Verbose && r.Extra == "missed" {
+		statusText = "Missed"
+	}
+
+	if _, err := fmt.Fprintf(buf, "%s: %s (Status: %d) [Size: %d]\n", statusText, r.Entity, r.Status, *r.Size); err != nil {
```

Comment:
I think it'd be better if:

* `Extra` is left alone in the result.
* The result is always passed through when Verbose is set.
* Any result is passed through with the `Status` set.
* The handler looks at the status to see if it's good or bad.

Verbose mode should also indicate "same as base" or something if the vhost responds with the same content.

Thoughts?

Reply:
The `Status` field is the return code which does not indicate success or failure (at least in this mode). I was thinking about adding a new field to the `Result` object indicating the result is a verbose one or smth like that.
What do you mean by `is passed through`?

Reply:
@OJ ok now I got what you meant. Implemented it for VHOSTS for now. Once this is merged we can update the other modes too to use the new (better) way